geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -225,6 +225,25 @@ <h2 class="sr-only">
                     {% endif %}
                 </dd>
             {% endif %}
+            {% if document.material_support %}
+                {% regroup document.material_support by material_support as ms_list %}
```

Comment:
```suggestion
            {% if document.fragments_by_material_support %}
                {% regroup document.fragments_by_material_support by material_support as ms_list %}
```

This should call the new `fragments_by_material_support` property you added to `Document`, otherwise all working well locally!